tests/projects/simple-bios-shell/hie.yaml

```
@@ -8,3 +8,7 @@ cradle:
       ECHO "-Wall" >> %HIE_BIOS_OUTPUT%
       ECHO "A" >> %HIE_BIOS_OUTPUT%
       ECHO "B" >> %HIE_BIOS_OUTPUT%
+    dependency-shell: |
+      echo -n # > nul && ECHO hie.yaml >> %HIE_BIOS_OUTPUT%
```

```suggestion
      echo -n # > nul && ECHO hie.yaml>> %HIE_BIOS_OUTPUT%
```
watching the logs only left this little step, thanks for take care 😃